src/pages/industries/financial-services.jsx

```diff
@@ -180,6 +181,15 @@ const financialResources = [
     description:
       "Robinhood's war stories from running Cilium in a high-churn near-production environment, learn how they overcame challenges by better understanding and tuning Cilium.",
   },
+  {
+    imageSrc: SicrediOfficeImage,
+    imageAlt: 'Sicredi logo on a wall',
+    title: 'Strengthening Security Across Distributed Kubernetes Clusters',
+    buttonLink: 'https://www.youtube.com/watch?v=MSqI-gBiCrc',
+    buttonText: 'Watch the Talk',
+    description:
+      "Sicredi, Brazil's largest credit union, leveraged Cilium to implement a full security policy across its infrastructure.",
```

Comment:
What does this mean?

Reply:
Sicredi used Cilium cluster mesh mainly for service mesh, cluster mesh and security that came with it.  I wanted to compress all that detail into a one liner and also got that from their talk description at kubecon